src/app/features/search/search.component.spec.ts

```diff
@@ -78,14 +76,46 @@ describe('SearchComponent', () => {
 
         expect(navigateSpy).toHaveBeenCalledWith(['/search/results'], { queryParams: { trailerId: expectedTrailerId } });
       });
+    });
 
-      it('should add error', () => {
+    describe('invalid input', () => {
+      it('should add search-term error', () => {
         const addErrorSpy = jest.spyOn(globalErrorService, 'addError').mockImplementation(() => {});
 
         component.navigateSearch('', '');
 
         expect(addErrorSpy).toHaveBeenCalledWith({ error: component.missingTermErrorMessage, anchorLink: 'search-term' });
       });
-    });
+
+      it('should add search-type error', () => {
+        const addErrorSpy = jest.spyOn(globalErrorService, 'addError').mockImplementation(() => {});
+
+        component.navigateSearch('some term', '');
+
+        expect(addErrorSpy).toHaveBeenCalledWith({ error: component.missingTypeErrorMessage, anchorLink: 'search-type' });
+      });
+    })
+
+    describe('helper methods', () => {
+      it('should get inline error message', () => {
```

Comment:
Would this test be better if you mocked the return value of `this.globalErrorService.errors$` and then test that it returns the expected boolean value?

Reply:
Great spot! And sorry, indeed this test has no value as it is now. Suggestion applied.